### PR TITLE
reorganize top-level

### PR DESCRIPTION
**Pull request by ltalirz**
* reorganize top-level of docs
 * use pydata layout (as on aiida-core)
 * add logo
 * add "Edit on Github"  button
 * add prominent link back to materialscloud.org/aiidalab
 * speed up running locally by using dockerhub image by default

Todo:
 - [ ] make sure this link redirects https://aiidalab.readthedocs.io/en/latest/deployment/index.html

### CODE REVIEW

**Comment by ltalirz** [comment]
@csadorf I suggest you start by building the docs locally and getting an impression of the overall organisation

File: docs/source/usage/index.rst [hunk 1]
```diff
@@ -1,8 +1,50 @@
...
 
 
+***************
+Access AiiDAlab
+***************
+
+As a user, you have two options to access AiiDAlab:
+
+ 1. Log into one of the `open AiiDAlab servers <https://materialscloud.org/aiidalab>`_.
+ 2. Run the AiiDAlab docker container on your local machine.
```
Comment (csadorf) [inline docs/source/usage/index.rst:13]: What about the option to just run the image from the docker hub?

**Review by csadorf — APPROVED** [review]
One question, but looks good!

File: docs/source/usage/index.rst (hunk 1 above)
Comment (ltalirz) [reply docs/source/usage/index.rst:13]: Thanks for the quick review!

> What about the option to just run the image from the docker hub?

That would fall under 2. ; I haven't yet adapted the instructions below to start with this but I'll do it now

Comment (ltalirz) [reply docs/source/usage/index.rst:13]: added - I'll go ahead with the merge

**Pull request merged**